config.js

```diff
@@ -32,6 +32,9 @@ var config = {
         host: '0.0.0.0'
     },
     authentication: {
+        enabled: true,
```

Comment:
Two questions with regards the new parameters:

* There should be an environment variable corresponding to each one of them (properly processed in the code and documented). Have they been included?
* Thinkinig in backward compatilbity, which are the default vlues for the 3 new elements in the case they doesn't appear in config.js/envars?

Reply:
hi @fgalan auth parameters where not handled as env variables before, so i didn't work that out.

regarding backward compatibility, given that url when missing is computed form host and port that would not be an issue. also, if enabled and type not specified, assumption is that it is 'keystone' the provider used. PS: enabled was already there, but not documented.

Reply:
ps: also header default value is the old one and is loaded from a new constants

Reply:
as regards pre-existence of `enabled` see: https://github.com/telefonicaid/iotagent-node-lib/pull/591/files#diff-31cabb4f8b54ad34fdf6b4b99d056f49L346

Reply:
1. let me know if current backward compatibility is fine or any changes are needed
1. let me know if despite before env variables where not used for authentication you want me to introduce them.

Reply:
1. According to your information, backward compatibility seems fine.
2. No need to add env var (at least in the context of this PR) if they have been never used for this kind of configuration.

NTC